metaflow/plugins/resources_decorator.py

```diff
@@ -33,4 +33,10 @@ class ResourcesDecorator(StepDecorator):
     """
```

Comment:
```
disk : int, default: 10240
   Disk size (in MB) required for this step. Only applies on Kubernetes.
```

Reply:
could you add this to the docstring for the decorator for clarity? otherwise the PR is looking great :)